KProcessHacker/KProcessHacker.vcxproj

```diff
@@ -51,6 +52,7 @@
     <PlatformToolset>WindowsKernelModeDriver10.0</PlatformToolset>
     <ConfigurationType>Driver</ConfigurationType>
     <DriverType>WDM</DriverType>
+    <Driver_SpectreMitigation>Spectre</Driver_SpectreMitigation>
```

Comment:
👍 - I already have this in a the private branch for the hardening, happy to see this get in now.